src/D2L.Bmx/Program.cs

```diff
@@ -210,25 +210,30 @@
 // start bmx
 return await new CommandLineBuilder( rootCommand )
 	.UseDefaults()
-	.AddMiddleware( async (
-		context,
-		next
-	) => {
-		await UpdateChecker.CheckForUpdatesAsync( configProvider.GetConfiguration() );
-		//Creating new Cache directory and removing old files.
-		if( !Directory.Exists( BmxPaths.CACHE_DIR ) ) {
-			try {
-				Directory.CreateDirectory( BmxPaths.CACHE_DIR );
-				File.Delete( Path.Join( BmxPaths.BMX_DIR, "awsCredsCache" ) );
-				File.Delete( Path.Join( BmxPaths.BMX_DIR, "sessions" ) );
-				File.Delete( Path.Join( BmxPaths.BMX_DIR, "update_check" ) );
```

Comment:
`awsCredsCache` and `update_check` are never released, so there's no need to handle them.

Moving (rather than deleting) `sessions` keeps user's old cached session, which is better UX.

Reply:
Sorry, what does 'released' mean here? I'm having trouble finding it on google because of the word 😅

Reply:
I mean they're not in bmx v3.0.0, so "the people" never got them.

Reply:
Ah right. Thanks Chenfeng